lex.dd

```diff
@@ -746,24 +735,23 @@ $(GNAME HexLetter):
     $(B D)
     $(B E)
     $(B F)
-    $(D _)
 )
 
-	$(P Integers can be specified in decimal, binary, octal, or hexadecimal.)
+	$(P Integers can be specified in decimal, binary or hexadecimal.)
```

Comment:
I think on the "Files changed" tab you can click the pencil icon (top right corner of the source code section) to make further changes.